### DISCUSSION TITLE
Missing Observable imports in documentation and in Stackblitz example

### DISCUSSION THREAD

**Discussion by devzom** [ReactiveX/rxjs#6904] [Report issues other than bug]
Hey, I've noticed that the examples of [observable docs](https://rxjs.dev/guide/observable)  in section: **Disposing Observable Executions** all are missing 
`import { Observable } from 'rxjs';` the same in the Stackblitz examples.

**Comment by jakovljevic-mladen** [comment]
Hi @devzom, I proposed a fixing PR: #7026. Thank you for reporting.

EDIT: This is now merged to master, locking the issue.

**Reply by devzom** [reply]
Great, thank You :)
